src/data/endpoints/linodes.yaml

```diff
@@ -655,7 +655,8 @@ endpoints:
     type: Action
     authenticated: true
     description: >
-      Changes a Linode's hypervisor from Xen to KVM.
+      Changes a Linode's hypervisor from Xen to KVM. This endpoint can only be
+      used if your Linode is currently Xen and you wish to move to KVM.
```

Comment:
Might be worth including more info on what KVM and Xen are, and why the upgrade is desirable?  Something like 

> Changes a Linode's hypervisor from Xen (legacy) to KVM (modern).  Doing this restarts and migrates the Linode, and can take several minutes depending on the size of the Linode.  Upgrading to KVM has significant performances improvements, including a 300% boost in Unixbench scores and faster boots and shutdowns.  This endpoint will only work for Linodes running on the Xen hypervisor.